test/index.js

```diff
@@ -71,22 +71,114 @@ var testCases = [
   },
   {
     mesg: 'deals with duplicates correctly',
-    text: 'duplicate',
-    slug: 'duplicate'
+    text: 'duplicates',
+    slug: 'duplicates'
   },
   {
     mesg: 'deals with duplicates correctly-1',
-    text: 'duplicate',
-    slug: 'duplicate-1'
+    text: 'duplicates',
+    slug: 'duplicates-1'
   },
   {
     mesg: 'deals with duplicates correctly-2',
-    text: 'duplicate',
-    slug: 'duplicate-2'
+    text: 'duplicates',
+    slug: 'duplicates-2'
   },
```

Comment:
The tests cases later also contained `duplicate`, to ensure those tests are in sync with the referred repo, I changes these here instead.